### DISCUSSION TITLE
makeSchema help when using sourceTypes with the generated prisma file  - TypeError:..\src\prisma\client\index.d.ts: Property right of AssignmentExpression expected node to be of a type ["Expression"] but instead got undefined

### DISCUSSION THREAD

**Discussion by bustinjurkz** [graphql-nexus/nexus#962] [Q&A]
Hello, I'm trying to NOT use nexus prisma plugin, and the imported index.d.ts file which prisma generates is problematic.  

`  sourceTypes: {
    modules: [
      {
        module: require.resolve('./prisma/client/index.d.ts'),
        alias: 'prisma',
      },
    ],
  },`
  
  When I run the server, in order to hopefully get types in my schema file,  I then get this error : TypeError:..\src\prisma\client\index.d.ts: Property right of AssignmentExpression expected node to be of a type ["Expression"] but instead got undefined.
  
  Has anyone encountered this before? I am using the latest TypeScript, Nextjs, prisma, and nexus, with node v14.16.9.
  
  
  
  The schema generated on prisma's end successfully.  I can send the schema if needed.

**Comment by nenadfilipovic** [comment]
Module already uses resolve, you just have to type name of package or path. For prisma it should look like this `module: '.prisma/client'`
